.github/CODEOWNERS

```
@@ -1,6 +1,6 @@
 *       @Jigsaw-Code/outline-dev
 
-/cordova-plugin-outline/  @Jigsaw-Code/outline-networking-owners
+/src/cordova/plugin/  @Jigsaw-Code/outline-networking-owners
 /third_party/             @Jigsaw-Code/outline-networking-owners
 /tools/                   @Jigsaw-Code/outline-networking-owners
```

alignment thing as mentioned above. feel free to ignore this comment if it's just a display issue.

```suggestion
/tools/               @Jigsaw-Code/outline-networking-owners
```